corehq/apps/domain/templates/domain/email/domain_transfer_confirm.html

```diff
@@ -0,0 +1,8 @@
+Initiator: {{ from_username }}
+Project Space: {{ domain }}
+Acceptor: {{ to_username }}
+
+Please validate that the transfer is valid and email both parties notifying them that the transfer is complete.
```

Comment:
I just copied and pasted this from a spec and only proofread for egregious grammar errors